dataset/dictionary/y-events.json5

```diff
@@ -186,6 +186,151 @@
     tags: [ "event", "character-sub" ],
   },
 
+  // v4.5
+  {
+    en: "Blades Weaving Betwixt Brocade",
+    ja: "鋒刃、錦を裁つ",
+    zhCN: "铓锋锦间裁",
+    pronunciationJa: "ほうじんにしきをたつ",
+    tags: [ "event" ],
+    notes: "v4.5 期間限定イベント",
```

Comment:
Not event name, just code name only

Reply:
Attached is the revised document.
problem
```
    notes: "v4.5 期間限定イベント",
```
after
```
    notes: "v4.5 バージョンサブタイトル",
```